f5-distributed-cloud/manifest.json

```
@@ -0,0 +1,54 @@
+{
+   "manifest_version": "2.0.0",
+    "app_uuid": "74c33838-0310-4ef3-95db-c378aece9d8b",
+    "app_id": "f5-distributed-cloud-services",
+    "display_on_public_website": true,
+    "tile": {
+      "overview": "README.md#Overview",
+      "configuration": "README.md#Setup",
+      "support": "README.md#Support",
+      "changelog": "CHANGELOG.md",
+      "description": "Stream and visualize F5 Distributed Cloud Services event logs.",
+      "title": "F5 Distributed Cloud Services",
+      "media": [
+        {
+          "media_type": "video",
+          "caption": "Configuring remote logging from F5 Distributed Cloud Services to Datadog",
+          "image_url": "https://youtu.be/oONsA-k7n5s"
+        },
+        {
+          "media_type": "image",
+          "caption": "The F5 Distributed Cloud Services global log receiver",
+          "image_url": "images/logreceiver_config.png"
```

```suggestion
          "image_url": "images/logreceiver-config.png"
```

Or you can rename the file